public/index.js

```diff
@@ -64,8 +64,12 @@ buttons.forEach((button) => {
       .then((response) => response.json())
       .then((data) => {
         if (resultElement) {
-          resultElement.innerHTML = `<p>${data.choices[0].message.content}</p>`;
-          
+          const textContent = data.text.choices[0].message.content;
+          const imageUrl = data.image.data[0].url;
+          resultElement.innerHTML = `
+            <p>${textContent}</p>
+            <img src="${imageUrl}" alt="Generated Image">
```

Comment:
It's currently not tidy or well formatted. The picture returns far too oversized. I will fix it this week. We can then look at re-locating the image to background. @josueJURE

Reply:
I think I know how to relocate the image to background. I'm going to create a branch and give it a shot